orchestra/tests/helpers/fixtures.py

```diff
@@ -245,4 +251,52 @@ def setup_models(test_case):
                 task=task,
                 status=assignment_status,
                 assignment_counter=i,
-                in_progress_task_data=task_data)
+                in_progress_task_data=task_data,
+                start_datetime=parse(
+                    '2015-10-12T0{}:00:00+00:00'.format(2 + i)))
+
+
+def task_back_and_forth(test):
+    task = test.tasks['rejected_review']
```

Comment:
should the function take `task_slug` as an argument, or will we always use `'rejected_review'`?